src/w3c/seo.js

```diff
@@ -11,7 +11,7 @@ export function run(conf) {
   switch (conf.canonicalURI) {
     case "edDraft":
       if (conf.edDraftURI) {
-        conf.canonicalURI = new URL(conf.edDraftURI, document.location).href;
```

Comment:
Would you run `npm run lint -- --fix` and commit the result? Thanks!

Reply:
Done